frontend/src/components/DataHelpers.tsx

```diff
@@ -1,43 +1,28 @@
-import Button from 'react-bootstrap/Button';
+import { ButtonGroup, Button }  from 'react-bootstrap';
 
 import ItemType from '../types/ItemType';
 
 interface DataHelpersProps {
     items: ItemType[];
     setItems: any;
+    services: any;
 };
 
-const DataHelpers = ({items, setItems}:DataHelpersProps) => {
+const DataHelpers = ({items, setItems, services}:DataHelpersProps) => {
 
-    let BACKEND_URL = process.env.REACT_APP_BACKEND_URL || 'localhost:8000';
+    let BACKEND_URL:string = process.env.REACT_APP_BACKEND_URL || 'localhost:8000';
     // console.log(process.env, BACKEND_URL);
     BACKEND_URL = 'http://' + BACKEND_URL;
     //'http://todo-backend-yvakil-backend.apps.na46.prod.nextcle.com/'; // http://localhost:8000';
 
-    const postToEndpoint = () => {
-        const request = {
-            method: 'POST',
-            headers: { 'Content-Type': 'application/json'},
-            body: JSON.stringify(items)
-        };
-
-        fetch(BACKEND_URL, request)
-            .then(response => response.json())
-            .then(data => console.log(data));
-    };
-
-    const getFromEndpoint = () => {
-        fetch(BACKEND_URL)
-            .then(response => response.json())
-            .then(data => setItems(data));
-    };
-
     return (
-        <>
-            <Button onClick={()=>setItems([])}>Clear All</Button> 
-            <Button onClick={()=>postToEndpoint()}>Save to DB</Button> 
-            <Button onClick={()=>getFromEndpoint()}>Get from DB</Button>
-        </>
+        <div>
+            <ButtonGroup>
+                <Button onClick={()=>setItems([])}>Clear All</Button> 
+                <Button onClick={()=>services.postToEndpoint(BACKEND_URL, items)}>Save to DB</Button> 
+                <Button onClick={()=>services.getFromEndpoint(BACKEND_URL, setItems)}>Get from DB</Button>
```

Comment:
I think these functions could be provided to this component directly, without having to pass items. Then this component could be used like:
```jsx
<DataHelpers
    onClick={() => setItems([])}
    onGet={() => services.postToEndpoint(BACKEND_URL, items)}
    onPost={() => services.getFromEndpoint(BACKEND_URL, setItems)}
/>
```